modules/ocf_printhost/files/ocfps

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Use the user command pdftops, not the CUPS filter.
+PDFTOPS=/usr/bin/pdftops
+
+if [ "$#" -ne 5 ] && [ "$#" -ne 6 ]; then
+    echo "Usage: $0 job-id user title copies options [file]"
+    exit
+fi
+
+if [ "$#" -eq 5 ]; then
+    $PDFTOPS - - | pstops "$@"
+else
+    $PDFTOPS "$6" - | pstops "${@:1:5}"
+fi
```

Comment:
Maybe include some of that in a comment, or at least the part about the arguments, because that's the important part in this section. Seems better than putting all that in documentation, the overall process and architecture should be in the documentation, but specifics like arguments and stuff seem like they belong here more.